CONTRIBUTING.md

```diff
@@ -1,3 +1,9 @@
+## Getting Started
+
+Install dependencies: `npm ci`
+
+Useful commands: `npm start` (dev server), `npm run build`, `npm test`, `npm run lint`
```

Comment:
We already have quite some steps in the README under [getting-started-with-development](https://github.com/bitmovin/bitmovin-player-ui?tab=readme-ov-file#getting-started-with-development). Why do we need this section here again? 🤔